exonum-java-binding/core/rust/src/storage/proof_map_index.rs

```diff
@@ -38,9 +38,32 @@ use storage::{
 use utils;
 use JniResult;
 
-type Key = [u8; PROOF_MAP_KEY_SIZE];
+#[derive(Clone, Default)]
+struct Key([u8; PROOF_MAP_KEY_SIZE]);
 type Index<T> = ProofMapIndex<T, Key, Value>;
 
+impl ObjectHash for Key {
+    fn object_hash(&self) -> Hash {
+        hash(&self.0)
```

Comment:
This changes the only currently enabled mode (no hashing to hashing), breaking it. Why is that?

Reply:
was fixed, actually `Hash::from_slice` is used

Reply:
This kind of change is provided in the #1245 which you probably were not aware of because you're not in the reviewers' list...